syncr_backend/run_backend.py

```diff
@@ -1,31 +1,79 @@
 #!/usr/bin/env python
 import argparse
+import sys
+import threading
+from typing import Any
 from typing import List
 
-from syncr_backend import drop_init
-from syncr_backend import node_init
+from syncr_backend.init import drop_init
+from syncr_backend.init import node_init
+from syncr_backend.network import send_requests
+from syncr_backend.network.listen_requests import listen_requests
 
 
-def execute_node_function(function_name: str, args: List[str]):
+def run_backend() -> None:
     """
-    Runs a function with the given args
+    Runs the backend
+    """
+    input_args_parser = argparse.ArgumentParser()
+    input_args_parser.add_argument(
+        "ip",
+        type=str,
+        nargs=1,
+        help='ip to bind listening server',
+    )
+    input_args_parser.add_argument(
+        "port",
+        type=str,
+        nargs=1,
+        help='ip to bind listening server',
+    )
+    arguments = input_args_parser.parse_args()
 
-    :param function_name: string name of the function to run
-    :param args: arguments for the function to run
+    request_listen_thread = threading.Thread(
```

Comment:
Can we add an optional `mode` option, one for interactive and one for daemon? And do this for interactive and for daemon just run the server